### DISCUSSION TITLE
Current Date & time in tab

### DISCUSSION THREAD

**Discussion by mat-m** [hluk/CopyQ#2059] [Q&A]
I saw the command to have a global command here: https://copyq.readthedocs.io/en/latest/command-examples.html#paste-current-date-and-time

Is there a way to insert the output in a specific position in a tab ? My goal is to have it available for paste whenever I choose this tab, but to not have a big bunch of timestamps in this tab.

Any help welcome

**Comment by dbareis** [comment]
@mat-m 
Sorry but I can't make out what you want or what you mean by a tab, the sample requires you to position the cursor and press the specified hotkey.

**Comment by mat-m** [comment]
These are tabs:
![image](https://user-images.githubusercontent.com/1039753/188007897-53c5d36e-3892-4c19-b2a2-177c1395f60c.png)

I get them through a shortcut, and they display as a menu; The first one is the clipboard history, so it appears as a menu with the search field in top position.
The second is a permanent items list that I want always available.

1. Can we populate the clipboard history with the timestamp without actually pasting it where the cursor is ?
2. Can we have an entry in the "permanent" list to have the same effect as the hotkey you mention (paste at the cursor position) ,

Thank you

**Reply by dbareis** [reply]
I think you want to add teh date and time to a tab and the "write" function does that: https://copyq.readthedocs.io/en/latest/scripting-api.html#write

I don't know what a "permanent list" is.

**Reply by mat-m** [reply]
I have a tab where there are only fixed items, and no clipboard dynamic inputs.
I would like to have a command in this tab to paste the current timestamp.
So when I call the tab with a global shortcut, I just have toi click the entry and the current timestamp is pasted.

I managed to do it with a global shortcut, but not set it in a tab.
I found some action dialog to have the function, but I can't find a way to have that available in the tab:

![image](https://user-images.githubusercontent.com/1039753/207183001-4213397a-9cdb-4867-8610-eac02be29ca5.png)

If I click apply, it inserts a static timestamp.
